packages/leancode_lint/lib/lints/bloc_const_constructors.dart

```diff
@@ -0,0 +1,46 @@
+import 'package:analyzer/error/error.dart' hide LintCode;
+import 'package:analyzer/error/listener.dart';
+import 'package:custom_lint_builder/custom_lint_builder.dart';
+import 'package:leancode_lint/helpers.dart';
+
+class BlocConstConstructors extends DartLintRule {
```

Comment:
What do you think about adding DartFix for this lint? Lint that will create a const constructor. 
It could be problematic if Bloc has variables, but I think we have to only handle case where there are no parameters in Bloc's body. And in this case, we only have to create an empty const constructor